.evergreen/.evg.yml

```diff
@@ -2121,15 +2149,26 @@ buildvariants:
   display_name: "${version} ${compressor} ${topology} ${auth} ${ssl} ${jdk} ${os} "
   tags: ["tests-variant"]
   tasks:
-    - name: "test"
+    - name: "test-sync"
+    - name: "test-reactive"
+    - name: "test-core-and-legacy"
+
+- matrix_name: "tests-unit"
```

Comment:
Just run unit tests once on each JDK, instead of with every topology and release of MongoDB>